airbyte-integrations/connectors/source-outreach/source_outreach/manifest.yaml

```diff
@@ -0,0 +1,5755 @@
+    $parameters:
+      path: "/personas"
+
+  - $ref: "#/definitions/base_stream"
+    name: "prospects"
+    primary_key: "id"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/prospects_schema"
+    $parameters:
+      path: "/prospects"
+
+  - $ref: "#/definitions/base_stream"
+    name: "sequences"
+    primary_key: "id"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/sequences_schema"
+    $parameters:
+      path: "/sequences"
+
+  - $ref: "#/definitions/base_stream"
+    name: "sequence_states"
+    primary_key: "id"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/sequence_states_schema"
+    $parameters:
+      path: "/sequenceStates"
+
+  - $ref: "#/definitions/base_stream"
+    name: "sequence_steps"
+    primary_key: "id"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/sequence_steps_schema"
+    $parameters:
+      path: "/sequenceSteps"
+
+  - $ref: "#/definitions/base_stream"
+    name: "snippets"
+    primary_key: "id"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/snippets_schema"
+    $parameters:
+      path: "/snippets"
+
+  - $ref: "#/definitions/base_stream"
+    name: "stages"
+    primary_key: "id"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/stages_schema"
+    $parameters:
+      path: "/stages"
+
+  - $ref: "#/definitions/base_stream"
+    name: "tasks"
+    primary_key: "id"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/tasks_schema"
+    $parameters:
+      path: "/tasks"
+
+  - $ref: "#/definitions/base_stream"
+    name: "templates"
+    primary_key: "id"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/templates_schema"
+    $parameters:
+      path: "/templates"
+
+  - $ref: "#/definitions/base_stream"
+    name: "users"
+    primary_key: "id"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/users_schema"
+    $parameters:
+      path: "/users"
+
+definitions:
+  selector:
+    type: RecordSelector
+    extractor:
+      type: CustomRecordExtractor
+      class_name: source_outreach.components.CustomExtractor
+
+  requester:
+    type: HttpRequester
+    url_base: https://api.outreach.io/api/v2
+    http_method: GET
+    request_parameters:
+      count: "false"
+      sort: updatedAt
+      page[size]: "1000"
+    request_headers: {}
+    authenticator:
+      type: OAuthAuthenticator
+      client_id: "{{ config['client_id'] }}"
+      client_secret: "{{ config['client_secret'] }}"
+      refresh_token: "{{ config['refresh_token'] }}"
+      token_refresh_endpoint: https://api.outreach.io/oauth/token
+      grant_type: refresh_token
+      scopes:
```

Comment:
Please delete the scopes. Having all scopes is not necessary to sync a specific stream. By default, the availability check should filter the stream if it's not accessible to the user.

Reply:
The scopes are given according to the credential, which are the scopes to be deleted?

Reply:
@lazebnyi

Reply:
All, as I said if user what to sync one stream he no need grant all scopes.

Reply:
Done!